torchao/prototype/low_bit_optim/subclass_4bit.py

```diff
@@ -31,17 +32,29 @@ def __new__(cls, codes: Tensor, scale: Tensor, qmap: Tensor, signed: bool, shape
         )
 
     def __init__(self, codes: Tensor, scale: Tensor, qmap: Tensor, signed: bool, shape):
+        """Create quantized 4-bit optimizer state as proposed in https://arxiv.org/abs/2309.01507.
```

Comment:
btw the link is not valid, can you remove `.` in the end?

Reply:
Done